tests/chainer_tests/functions_tests/math_tests/test_sqrt.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class UnaryFunctionsTestBase(unittest.TestCase):
```

Comment:
And I think of providing abstraction for lifting basic numpy/cupy functions to Chainer function, now there are many similar codes for them as you did for trigonometric functions.